pegasus/sites.v3/hourofcode.com/public/css/page/how-to.scss

```diff
@@ -79,6 +79,10 @@ nav.choose-guide {
       background: var(--neutral_white);
       border-color: var(--neutral_white);
     }
+
+    &:is(:hover, :active):not(.active) {
+      background: rgba(255, 255, 255, 0.4);
+    }
```

Comment:
Sneaking these How-to guide hover state styles into this PR: 

https://github.com/user-attachments/assets/224ff0ee-5a01-401d-abf1-02ddebb6f435